user-service/src/main/java/org/arpha/security/jwt/AuthJwtTokenFilter.java

```diff
@@ -37,14 +37,13 @@ public AuthJwtTokenFilter(JwtUtils jwtUtils, UserDetailsService userDetailsServi
     @Override
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
         if (!hasAuthorizationBearer(request)) {
-            filterChain.doFilter(request, response);
-            return;
+            throw new InternalAuthorizationException("Authorization token is missing!");
         }
+
         String token = getAccessToken(request);
 
         if (!jwtUtils.isTokenValid(token)) {
-            filterChain.doFilter(request, response);
-            return;
+            throw new InternalAuthorizationException("Token is not valid or expired!");
```

Comment:
as far as I know it's not a very good idea to throw exception in filter shouldn't we let filterS finish their work?